assembl/graphql/vote_session.py

```diff
@@ -509,6 +576,16 @@ def mutate(root, args, context, info):
                 nb_ticks=args['nb_ticks'],
                 unit=args['unit']
             )
+            proposal_id = args.get('proposal_id')
```

Comment:
This copy/pasta can become a method. Repeated across multiple mutations.

Reply:
Which copy pasta are you talking about here, args.get?